### PR TITLE
Support a `prodperfect_test` cookie

### PR DESCRIPTION
**Pull request by chanson**

### CODE REVIEW

**Comment by chanson** [comment]
With Data:


Without Data:

**Comment by funkhouserw** [comment]
Looks good as far as I'm concerned!  :shipit:

File: test/testcafe/regression-tests.js [hunk 1]
```diff
@@ -1,12 +1,35 @@
...
 
-
+let testSuiteRunId = undefined;
 const getLocation = ClientFunction(() => document.location.href);
 const script_loader_function = new Function(fs.readFileSync('./test/testcafe/test-snippet.js').toString());
+const setProdPerfectCookie = ClientFunction( (id, name, testSuiteRunId, env) => {
+  const data = {
+    test_run_data: {
+      test_run_id: id,
+      test_suite_id: testSuiteRunId,
```
Comment (danwiding) [inline test/testcafe/regression-tests.js]: this is confusing, is it an identifier for the suite or the run id? and how is it different from the line above

Comment (chanson) [reply test/testcafe/regression-tests.js]: sorry, should have been `test_suite_run_id`, indicating the entireity of the run, whereas the line above is for one test in particular

Comment (danwiding) [reply test/testcafe/regression-tests.js]: excellent - let's do that, and lets' call the line above it test_script_run_id

**Pull request merged**